app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_of_words = command.split()
+    if len(list_of_words) != 3:
+        print("Invalid command format. Please provide exactly 3 arguments.")
+        return
+
+    source_file, destination_file = list_of_words[1:]
+
+    if source_file == destination_file:
+        return
+    else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Please, remove the redundant `else`